seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/source/ConsumerMetadata.java

```diff
@@ -38,7 +38,7 @@ public class ConsumerMetadata implements Serializable {
     private String bootstrapServers;
     private Properties properties;
     private String consumerGroup;
-    private boolean commitOnCheckpoint = false;
+    private boolean commitOnCheckpoint = true;
```

Comment:
The default value need get from the `COMMIT_ON_CHECKPOINT`.

Reply:
We use `COMMIT_ON_CHECKPOINT` to define the default value.

```
    public static final Option<Boolean> COMMIT_ON_CHECKPOINT = Options.key("commit_on_checkpoint")
            .booleanType()
            .defaultValue(true)
            .withDescription("If true the consumer's offset will be periodically committed in the background.");
```

So, I think if user does not set it in config file, you need get the default value from `COMMIT_ON_CHECKPOINT.defaultValue()`. Otherwise, the defaultValue in Option will lose its meaning.

Reply:
ok, you're right. It's been changed.